### PR TITLE
kotlin taglib library module integrated

### PR DESCRIPTION
**Pull request by SomalRudra**
The Taglib library for kotlin is ready. It has three functions so far : 
1. to get the audiofie metadatas
2. to get the artWork
3. to update the tags
a corresponding javaClass and layout file to test the functions are ready.
The app compiles and builds fine but crashes at runtime. I'm creating a pull request. Please have a look at it.

### CODE REVIEW

File: ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.command.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Executable : /home/somal/Android/Sdk/cmake/3.10.2.4988404/bin/ninja
+arguments : 
+-C
+/home/somal/GSoCProgress/MuBandroidnewtry/musicbrainz-android/ktaglib/.cxx/cmake/debug/arm64-v8a
```
Comment (Miguel-Herrero) [inline ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.command.txt:4]: Maybe this `txt` with the path to your repo is not needed to be committed?

File: ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.stdout.txt [hunk 2]
```diff
@@ -0,0 +1,119 @@
+ninja: Entering directory `/home/somal/GSoCProgress/MuBandroidnewtry/musicbrainz-android/ktaglib/.cxx/cmake/debug/arm64-v8a'
```
Comment (Miguel-Herrero) [inline ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.stdout.txt:1]: Maybe this `stdout.txt` is not needed to be commited?

File: build.gradle [hunk 3]
```diff
@@ -23,6 +27,22 @@ allprojects {
     }
 }
 
-task clean(type: Delete) {
-    delete rootProject.buildDir
+//task clean(type: Delete) {
```
[the remark this answers is not in the document]
Comment (SomalRudra) [reply build.gradle:30]: A clean command for dir was already present.
So,I had to comment this out

File: ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.stdout.txt (hunk 2 above)
Comment (SomalRudra) [reply ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.stdout.txt:1]: no they are not required.
I will make the changes

File: ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.command.txt (hunk 1 above)
Comment (SomalRudra) [reply ktaglib/.cxx/cmake/debug/arm64-v8a/android_gradle_build_targets.command.txt:4]: no they are not required.
I will make the changes

**Comment by SomalRudra** [comment]
> Great work with the integration!!!
> 
> I'm a little lost with all the tag lib files, so my only question is “which files are really needed to be commited and which ones could be `.gitignored`”?

Thank you! 
please check tthis commit https://github.com/metabrainz/musicbrainz-android/pull/40/commits/09a5c7b10f8317ef916b22abb38d18d15fd08a26

File: app/build.gradle [hunk 4]
```diff
@@ -4,10 +4,9 @@ android {
     compileSdkVersion 29
     defaultConfig {
         applicationId "org.metabrainz.android"
-        minSdkVersion 15
+        minSdkVersion 21
```
[the remark this answers is not in the document]
Comment (amCap1712) [reply app/build.gradle:7]: I think we should not upgrade the minSdk yet. Let the app go into production and we can decide after a month or so.

**Comment by amCap1712** [comment]
@SomalRudra Since, we have tested that the module works. I suggest you replace taglib directory with jitpack. This will help in a better review. Also, there are some other files that can be gitignored but let's first do the jitpack replacement and then analyse the .gitignore files.

**Comment by SomalRudra** [comment]
> @SomalRudra Since, we have tested that the module works. I suggest you replace taglib directory with jitpack. This will help in a better review. Also, there are some other files that can be gitignored but let's first do the jitpack replacement and then analyse the .gitignore files.

okay! by today it's going to be done :)

**Pull request closed without merging**